{
	"schema_version": "1.6.0",
	"id": "UBUNTU-CVE-2014-1624",
	"modified": "2025-01-13T10:21:05Z",
	"published": "2014-01-28T00:55:00Z",
	"related": [
		"CVE-2014-1624"
	],
	"details": "Race condition in the xdg.BaseDirectory.get_runtime_dir function in python-xdg 0.25 allows local users to overwrite arbitrary files by pre-creating /tmp/pyxdg-runtime-dir-fallback-victim to point to a victim-owned location, then replacing it with a symlink to an attacker-controlled location once the get_runtime_dir function is called.",
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:14.04:LTS",
				"name": "pyxdg",
				"purl": "pkg:deb/ubuntu/pyxdg@0.25-4?arch=source&distro=trusty"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.25-4"
						}
					]
				}
			],
			"versions": [
				"0.25-3"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-xdg",
						"binary_version": "0.25-4"
					},
					{
						"binary_name": "python3-xdg",
						"binary_version": "0.25-4"
					}
				],
				"ubuntu_priority": "low"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2014/UBUNTU-CVE-2014-1624.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:16.04:LTS",
				"name": "pyxdg",
				"purl": "pkg:deb/ubuntu/pyxdg@0.25-4?arch=source&distro=xenial"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.25-4"
						}
					]
				}
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-xdg",
						"binary_version": "0.25-4"
					},
					{
						"binary_name": "python3-xdg",
						"binary_version": "0.25-4"
					}
				],
				"ubuntu_priority": "low"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2014/UBUNTU-CVE-2014-1624.json"
			}
		},
		{
			"package": {
				"ecosystem": "Ubuntu:18.04:LTS",
				"name": "pyxdg",
				"purl": "pkg:deb/ubuntu/pyxdg@0.25-4?arch=source&distro=bionic"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.25-4"
						}
					]
				}
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-xdg",
						"binary_version": "0.25-4"
					},
					{
						"binary_name": "python3-xdg",
						"binary_version": "0.25-4"
					}
				],
				"ubuntu_priority": "low"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2014/UBUNTU-CVE-2014-1624.json"
			}
		}
	],
	"references": [
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2014-1624"
		},
		{
			"type": "REPORT",
			"url": "https://www.cve.org/CVERecord?id=CVE-2014-1624"
		}
	]
}
